src/config/apolloclient.ts

```diff
@@ -143,6 +143,25 @@ const gqlClient = (navigate: any) => {
         authToken: getAuthSession('access_token'),
         userId: getUserSession('id'),
       },
+      retryAttempts: 5,
+      keepAlive: 30000,
+      shouldRetry: (error) => {
+        setLogs(`WebSocket retry attempt due to: ${error}`, 'info');
```

Comment:
Below seems to be reasonable suggestion to stop retrying incase of authentication errors.